src/js/init.s

```diff
@@ -118,6 +122,51 @@ SORT_START_ROW    = 3
 SORT_START_ROW    = 2
 .endif
 
+; .ifdef _CROWD_CONTROL_
+
+GAME_MODE_CHANGE_LOCATION = $01
+OP_JMP_ABS  = $4c
+
+; Short How-To-Use documentation:
+;
+; 0. Make sure that there isn't already anything waiting to be run by seeing if
+; CrowdControlFlag is zero. This gets cleared after the effects are completed.
+;
+; 1. Set one or more flags in the CrowdControlQueue field to run that operation
+; the next time the game is available.
+;
+; 2. If you need to update the status bar for an operation, set the appropriate bit
+; in CrowdControlStatusBarUpdate.
+;
+; 3. Set CrowdControlFlag to 1, and the game will run the update the next time the
+; player gets back to "normal" play state.
+;
+
+; If set, during the next main loop, run the next CC op from the queue
+; INTERNAL - bit 7 will be set when we need to early return (after wild warp)
+CrowdControlFlag              = $6220
```

Comment:
What is this RAM used for in vanilla?  It seems important.

Reply:
These are the bytes right after the NametableWriteBuffer, I *think* they are unused, but I don't have a solid way to check.

Reply:
Looks good, based on cov.ram (not in the repo, but I posted it in the discord).

I updated notes/ram with some extra information about various unused chunks.  If you wouldn't mind editing that file here to indicate how you're using the ram in your patch, that would be great.  Thanks!